src/discussions/posts/post/Post.jsx

```diff
@@ -35,6 +36,9 @@ function Post({
   const { reasonCodesEnabled } = useSelector(selectModerationSettings);
   const [isDeleting, showDeleteConfirmation, hideDeleteConfirmation] = useToggle(false);
   const [isClosing, showClosePostModal, hideClosePostModal] = useToggle(false);
+
+  const postURL = new URL(`${getConfig().PUBLIC_PATH}${courseId}/posts/${post.id}`, window.location.origin);
```

Comment:
I didn't get it, can you explain in detail?

The `PUBLIC_PATH` has a default value set to `'/'` ([source](https://github.com/openedx/configuration/blob/master/playbooks/roles/mfe/defaults/main.yml#L50)). It is used for *subdomain*-based deployments.
In the case of the *subfolder*-based deployments, it will be set for each MFE individually. In my case, its value is `'/discussions/'`.

The `BASE_URL` var has a form of the `"{{ MFE_BASE_SCHEMA }}://{{ MFE_BASE }}"` where the `MFE_BASE` will be 
the same value (`"mfe.{{ EDXAPP_LMS_BASE }}"`) for all MFEs in case of using *subdirectory*-based deployment (from [the README for the MFE deployer](https://github.com/openedx/configuration/blob/master/playbooks/roles/mfe_deployer/README.rst))

Why is the PUBLIC_PATH not valid?

How can I use the BASE_URL in case of *subdirectory*-based deployments?

Reply:
PUBLIC_PATH is actually used in the [discussions MFE](https://github.com/openedx/frontend-app-ora-grading/blob/master/src/data/constants/app.js#L3)

Reply:
here is the example.
`BASE_URL='http://localhost:2002'`
`${getConfig().BASE_URL}/${courseId}/topics/${post.topicId}`}`

Reply:
Please help me find out what I'm missing here because the solution looks correct to me

Reply:
> The PUBLIC_PATH has a default value set to '/' ([source](https://github.com/openedx/configuration/blob/master/playbooks/roles/mfe/defaults/main.yml#L50))

isn't it correct?

Reply:
I've tested my fix on devstack with default settings (no changes to the env were made) and it works as expected.

Here is another place where common config **always** has the PUBLIC_PATH: https://github.com/openedx/frontend-build/blob/master/config/webpack.prod.config.js#L57

Reply:
@mphilbrick211 could anyone help resolve this discussion?

Reply:
@dyudyunov the PUBLIC_PATH you are using is not defined in our production environment so we will not be able to merge this PR. Thank you for your suggestion.

Reply:
what if I use `${getConfig().PUBLIC_PATH || '/'}` instead?

Reply:
But I still don't understand why you're saying your instances are lacking the default for the PUBLIC_PATH (which is `'/'`) defined in both configuration and frontend-build repositories

Reply:
@asadazam93 and @muhammadadeeltajamul, any Open edX MFE must support path-based deployments, regardless of what your production deployment looks like. That's the whole point of the configuration variable: to allow users to deploy it in different ways.

With that in mind, it looks to me like @dyudyunov's fix is correct.

Reply:
@asadazam93 can you test it on the sandbox instance? I think you don't need to change your env because the default value (`'/'`) should work well for you